### PR TITLE
User Profile dynamic trip display(UI)

### PR DESCRIPTION
**Pull request by DivyaJyothiBN**

### CODE REVIEW

File: views/UserProfile.html [hunk 1]
```diff
@@ -37,212 +37,34 @@ <h1 class="brand cappbar-header no-hover fg-white mx-auto">Transport Management
...
         <script src="https://unpkg.com/leaflet@1.3.4/dist/leaflet.js" integrity="sha512-nMMmRyTVoLYqjP9hrbed9S+FzjZHW5gY1TWCHA5ckwXZBadntCNs8kEqAWdrb9O7rxbCaA4lKTIWjDXZxflOcA=="
-            crossorigin=""></script>
-		<script src="https://cdn.metroui.org.ua/v4/js/metro.min.js"></script>
-		<script src="https://code.jquery.com/jquery-3.3.1.min.js"></script>
+        crossorigin=""></script>
+        <script src="https://cdn.metroui.org.ua/v4/js/metro.min.js"></script>
+        <script src="https://code.jquery.com/jquery-3.3.1.min.js"></script>
         <script type="text/javascript" src="/js/userTripUpdate.js"></script>
         <script type="text/javascript" src="/js/staticMarker.js"></script>
+        <script>
```
Comment (DivyaJyothiBN) [inline views/UserProfile.html]: Will remove script

Comment (GaganCJ) [reply views/UserProfile.html]: Why? Which One?

File: public/js/userTripUpdate.js [hunk 2]
```diff
@@ -16,4 +17,65 @@ $(document).ready(function(){
...
+        },
+        {
+         date:"11/10/2019",
+        pickup:"pes",
+        drop:"living space girinagar",
+        time:"12:45"
+        }
+    ];
+    var trips= document.getElementById("trips");
+    for(var count=0; count<2;count++) {
```
Comment (GaganCJ) [inline public/js/userTripUpdate.js]: Can you optimize this code?

Comment (Gunashree-R-V) [reply public/js/userTripUpdate.js]: Which one?

**Pull request merged**